simulation_ws/src/cloudwatch_simulation/package.xml

```diff
@@ -16,8 +16,8 @@
   <depend>tf2</depend>
   <depend>gazebo_ros</depend>
   <depend>gazebo_plugins</depend>
-  <depend>robot_state_publisher</depend>
   <depend>nav2_bringup</depend>
+  <depend>turtlebot3_description_reduced_mesh</depend>
```

Comment:
why is ```turtlebot3_description``` required in simulation_ws now?

Reply:
see https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/blob/ros2/simulation_ws/src/cloudwatch_simulation/launch/turtlebot3_navigation.launch.py#L12

Reply:
see https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/blob/ros2/simulation_ws/src/cloudwatch_simulation/launch/bookstore_turtlebot_navigation.launch.py#L12

Reply:
https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/blob/ros2/simulation_ws/src/cloudwatch_simulation/launch/bookstore.launch.py#L27

Reply:
Ah okay it was there before this change.. I thought it's being added right now